Upstream Change I89ed102f: Drop NodeJS dependency in favor of
pure-python lesscpy https://review.openstack.org/#/c/37473/

Fixed in Havana 2013.2

--- horizon-2013.1.4/openstack_dashboard/settings.py.orig  2013-12-03 09:39:07.042100957 -0700
+++ horizon-2013.1.4/openstack_dashboard/settings.py 2013-12-03 09:39:34.835630769 -0700
@@ -112,9 +112,8 @@
     'django.contrib.staticfiles.finders.AppDirectoriesFinder',
 )

-less_binary = os.path.join(BIN_DIR, 'less', 'lessc')
 COMPRESS_PRECOMPILERS = (
-    ('text/less', (less_binary + ' {infile} {outfile}')),
+    ('text/less', ('lesscpy {infile}')),
 )

 COMPRESS_CSS_FILTERS = (
